tests/unit/ImplicitAnimationTests.swift

```diff
@@ -78,6 +78,9 @@ class ImplicitAnimationTests: XCTestCase {
     }
 
     XCTAssertEqual(addedAnimations.count, 1)
+    guard addedAnimations.count == 1 else {
+      return
```

Comment:
Should this include an `XCTAssertTrue(false)` to mark the test as failed?  Same question for the `guard`/`if-let` statements below.

Reply:
Similar to above, this is an early exit on the condition that our XCTAssertEqual assertion fails.

Reply:
Ah, I see them. Sorry&mdash;still finishing my coffee this morning...